config/routes.yaml

```diff
@@ -0,0 +1,11 @@
+article:
+    path: /articles/{id}
+    controller: libero.content_page.controller.content.scholarly_article
+    requirements:
+        id: '[A-Za-z0-9\-._]+'
+
+blog-article:
+    path: /blog/{id}
+    controller: libero.content_page.controller.content.blog_article
+    requirements:
+        id: '[A-Za-z0-9\-._]+'
```

Comment:
These should probably be enabled automatically based on configuration, otherwise has the same problem as https://github.com/libero/browser/pull/7#r210852139.